docs/developer_guide.md

```diff
@@ -90,6 +90,18 @@ The results will look somewhat like the following:
 opensearch-data-prepper   1.2.0-SNAPSHOT   3e81ef26250c   23 hours ago   566MB
 ```
 
+### Running smoke tests on a Docker Image
+
+To run automated smoke test on an image you can use the following command
+
+./release/smoke-tests/run-smoke-tests.sh -v _image_tag_ -r _image_repository_
```

Comment:
How will a developer know which of the following occurred?

* The tests succeeded
* The tests failed (the image was bad)
* The tests failed to start (maybe something wrong in their environment)

Reply:
I added additional logging messages and enhanced the readme to address these issues.